storage.go

```diff
@@ -369,5 +365,15 @@ func slicesEqual(s1, s2 []byte) bool {
 	return true
 }
 
+// dataKey generates a dataKey value based on the SKYDB_ENTROPY environment
+// variable.
+func dataKey() (crypto.Hash, error) {
+	ent, err := skydb.EntropyFromEnv()
+	if err != nil {
+		return crypto.Hash{}, err
+	}
+	return crypto.HashAll(ent, "dataKey"), nil
```

Comment:
Unless there is a reason why this can't be the case here, `crypto.Hash{}` would also be a valid datakey. The datakey doesn't necessarily need to be a random value.

Reply:
I'm not sure what the best approach is. I started with the public, private, and data key provided as configuration parameters. Karol expressed a preference to have just one value in the config, if possible, so I switched to using an entropy value from which I could derive the pub/sec key pair and the data key. I can use the entropy as data key but that doesn't sound like a good idea in case we need to save it somewhere. That's why I chose to derive a data key like this.

What do you think is the best approach? Use the entropy directly? Derive a data key in a different way? Hardcode a data key?

Reply:
From offline:  
Using an empty hash (`crypto.Hash{}`) is good enough here.